fink_filters/filter_kn_candidates/filter.py

```diff
@@ -0,0 +1,66 @@
+from pyspark.sql.functions import pandas_udf, PandasUDFType
+from pyspark.sql.types import BooleanType
+
+import pandas as pd
+
+@pandas_udf(BooleanType(), PandasUDFType.SCALAR)
+def kn_candidates(kn_score, drb, classtar, jd, jdstarthist, ndethist, cdsxmatch) -> pd.Series:
```

Comment:
The name of the input argument must match the name of a valid alert field. Here you just need to change `kn_score` with `knscore`. See the name declaration at:

https://github.com/astrolabsoftware/fink-broker/blob/132f4cf68e686cf0d0698c179021bb8b65072494/fink_broker/science.py#L179-L182

Reply:
This match is mandatory as the broker automatically searches for alert entries based on the constructor -- see:

https://github.com/astrolabsoftware/fink-broker/blob/132f4cf68e686cf0d0698c179021bb8b65072494/fink_broker/filters.py#L156-L245